scripts/numerics/visualize/visualizeDecodingRuntime.py

```diff
@@ -49,7 +51,6 @@ def runtime():
     yfinal = []
     cols = []
     colors = mcolors.BASE_COLORS
```

Comment:
This is apparently defined a second time down below. Consider removing the definition below. (see GitHub CodeQL warnings)